lib/node/node.ts

```diff
@@ -43,6 +43,12 @@ Zone.__load_patch('timers', (global: any, Zone: ZoneType, api: _ZonePrivate) =>
     // timers module not exists, for example, when we using nativescript
     // timers is not available
   }
+  if (isMix) {
+    // if we are in mix environment, such as Electron,
+    // the global.setTimeout has already been patched,
+    // so we just patch timers.setTimeout
+    return;
```

Comment:
@JiaLiPassion can you explain how global.setTimeout is patched already? I'm using Zone in mixed environment (Jest with jsdom), and I apparently need `global.setTimeout` to be patched, so `fakeAsync` from Angular test utils can work properly. But because it exits early, [this line](https://github.com/angular/zone.js/pull/774/files#diff-46621d39c83a1870b98f0b3e549daacaR56) is not reached.

This very line essentially breaks Angular testing with `fakeAsync` in Jest. We're tracking this under https://github.com/thymikee/jest-preset-angular/issues/65.

Any help appreciated.